include/io/fs/fs.h

```diff
@@ -24,7 +24,7 @@ namespace frontend {
 
 bool isFileExist(const std::string& filepath);
 void moveFile(const std::string& originFilename, const std::string& newFilename);
-void copyFile(const std::string& originFilename, const std::string& newFilename);
+void copyFile(const std::string& originFilename, const std::string& newFilename, const bool withDir);
```

Comment:
unexcepted const.
```suggestion
void copyFile(const std::string& originFilename, const std::string& newFilename, bool withDir);
```

or maybe
```suggestion
void copyFile(const std::string& originFilename, const std::string& newFilename, const bool& withDir);
```